lauramos/challenge_html_css/index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;300;700&display=swap" rel="stylesheet">
+    <link href="./src/fontawesome-free-5.15.1-web/css/fontawesome.css" rel="stylesheet">
+    <link href="./src/fontawesome-free-5.15.1-web/css/brands.css" rel="stylesheet">
+    <link href="./src/fontawesome-free-5.15.1-web/css/solid.css" rel="stylesheet">
```

Comment:
Podrias cambiar la implementacion para usar CDN y seria algo asi:

```suggestion
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/fontawesome.min.css" rel="stylesheet">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/brands.min.css" rel="stylesheet">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/solid.min.css" rel="stylesheet">
```

Reply:
Gracias Kriz, voy a estar sin acceso al pc desde ahorita, el martes lo reviso y lo arreglo de acuerdo a tus recomendaciones